docs/Four-Actions.md

```diff
@@ -7,3 +7,12 @@
 | Create An Advantage (Unknown Aspect) | Someone detrimental can reveal it to get the free invoke | The aspect remains unknown, but you get a Boost | Reveal the aspect and get *one* free invoke | Reveal the aspect and get *two* free invokes|
 | Attack | Attack is prevented | No Damage, Attacker Boost | Deal damage equal to shifts | Deal damage, and can trade 1 Shift for a Boost | 
 | Defend | Take damage / Attacker succeeds | Attacker creates a Boost | No Damage / Attacker fails | Prevent damage & *Defender* gains a Boost|
+
+## Weapon and Armor Interactions
+
+[Weapon and Armor](adopted-rules.md) do not impact Succeeding with Style, but their impact on damage can override the normal outcomes for Ties and Success. 
+
+| Action | Attack Failure | Attack Tie | Attack Succeed |
+|--------|------|------|------|
+| Weapon | Attack is prevented. Do not apply Weapon damage. | Apply only Weapon damage, but do not create a Boost. | Deal Shifts of damage, then add damage equal to Weapon Level. | 
+| Armor | No damage is taken. No Boosts are made.  | If Attacker has Weapon, subtract Armor from damage received. Otherwise, Attacker gets a Boost. | Subtract Armor level from damage. If damage is now 0, Defender Boost. |
```

Comment:
This isn't correct-- Attacker gets a boost if damage is reduced to 0, not the defender 🙂